frontend/test/metabase/scenarios/filters/relative-datetime.cy.spec.js

```diff
@@ -215,23 +215,21 @@ describe("scenarios > question > relative-datetime", () => {
     it("should allow changing values with starting from (metabase#22227)", () => {
       openOrdersTable();
 
-      cy.intercept("POST", "/api/dataset").as("dataset");
```

Comment:
`OpenOrdersTable` already comes with the intercept for the `POST /api/dataset` so there's no need to re-define it here.